dashboard/utils.py

```diff
@@ -372,6 +371,18 @@ def get_pearson_exam_status(self):  # pylint: disable=too-many-return-statements
             )
             return ExamProfile.PROFILE_INVALID
 
+    def course_has_exam(self, course_run):
+        """
+        Check if the user is authorized for an exam for a course run
+        """
+        return bool(
+            course_run and
+            course_run.edx_course_key and
+            course_run.course and
+            course_run.course.exam_module and
+            self.program.exam_series_code
+        )
```

Comment:
just to check if course has exam we need bool statement that will give us true or false